halo2-base/src/gates/range.rs

```diff
@@ -235,6 +295,9 @@ pub trait RangeInstructions<F: ScalarField> {
     }
 
     /// Returns `(c, r)` such that `a = b * c + r`.
+    /// * a: value to divide
+    /// * b: value to divide by
+    /// * a_num_bits: number of bits in `a`
```

Comment:
since `a` is variable, it's more accurate to say that we assume `a` has `<= a_num_bits` bits